src/interfaces/dAppSchema.ts

```diff
@@ -181,4 +181,5 @@ export interface DAppSchema {
   ownerAddress?: string;
   tokenId?: string;
   metrics?: object;
+  verification?: object;
```

Comment:
Just `object` is not sufficient. You need to have the full schema of verification also available.

Reply:
Yes I have added the Verification Schema.